views/networking/load_balancer/create.erb

```diff
@@ -65,27 +65,22 @@
                   }
                 ) %>
               </div>
-              <div class="col-span-6 md:col-span-2 xl:col-span-1">
-                <%== render(
-                  "components/form/text",
-                  locals: {
-                    name: "health_check_endpoint",
-                    label: "Health Check Endpoint",
-                    attributes: {
-                      required: true,
-                      placeholder: "/up"
-                    }
-                  }
-                ) %>
-              </div>
             </div>
+          </div>
+        </div>
+      </div>
+      <div class="px-4 py-5 mt-6 sm:p-6">
+        <div class="space-y-12">
+          <div>
+            <h2 class="text-base font-semibold leading-7 text-gray-900">Forwarding Rule</h2>
+            <p class="mt-1 text-sm leading-6 text-gray-600">Set how the traffic will be routed from the load balancer to your virtual machines.</p>
```

Comment:
Maybe: Configures the routing of traffic from the load balancer to your virtual machines.